docs/programming-oak.md

```diff
@@ -291,10 +291,11 @@ requests to it, over a channel that has end-to-end encryption into the Runtime
 instance:
```

Comment:
At some point the text above needs to be updated so it still describes the code below (i.e. includes a mention of the mysterious `label` parameter), but fine to defer.

Reply:
done.